tests/end2end/features/downloads.feature

```diff
@@ -337,6 +337,20 @@ Feature: Downloading things from a website.
         And I open the download with a placeholder
         Then "Opening *download.bin* with [*python*]" should be logged
 
+    Scenario: Opening a download with default-open-dispatcher set
+        When I set general -> default-open-dispatcher to python {} -c "import sys; print(sys.argv[1])"
+        And I open data/downloads/download.bin without waiting
+        And I wait until the download is finished
+        And I open the download with no args
```

Comment:
You don't need any python code for this, as you can just do `And I run :download-open`